ir.imansalmani.IPlan.json

```diff
@@ -35,7 +35,7 @@
         {
             "type" : "archive",
             "url" : "https://github.com/iman-salmani/iplan/releases/download/1.0.0/iplan-1.0.tar.gz",
-            "sha256" : "5165cb6a8b01a21613373e5e73b2ce39fac86348e8cbff50844082c613221301"
+            "sha256" : "5f312c2edfc2033c324db149814cd2ae286c4ab12a466c913ee29fa571cfc9e8"
```

Comment:
and as a side note, changing the archive without changing name is considered bad practice.